packages/common/src/services/resizer.service.ts

```diff
@@ -117,7 +118,7 @@ export class ResizerService {
       this._fixedWidth = fixedGridSizes.width;
     }
 
-    if (this.gridOptions) {
+    if (this.gridOptions && this.gridOptions.enableAutoResize) {
```

Comment:
but isn't this make the grid to not resize even with static sizes? 
Have you tried running the code in this lib project? 
We need to make sure that even if `enableAutoResize` is disabled, this won't break grids with static sizes like the Example 1 in this webpack demo.